spec/services/idv/steps/verify_step_spec.rb

```diff
@@ -11,19 +11,21 @@
       app_id: '123',
     )
   end
+  let(:request) do
+    double(
+      'request',
+      remote_ip: Faker::Internet.ip_v4_address,
+      headers: { 'X-Amzn-Trace-Id' => amzn_trace_id },
```

Comment:
just one spec error on spec/features/idv/doc_auth/verify_step_spec.rb / trace_id

Reply:
huh, this spec passes for me

Reply:
spec/services/idv/steps/verify_step_spec.rb worked, but its cousin in `features` spec/features/idv/doc_auth/verify_step_spec.rb was failing, fixed in 9e4acce4fe2b13bbceb66dab0b644a1b039784b0